Source/WebCore/css/CSSValueKeywords.in

```diff
@@ -1414,6 +1414,14 @@ spring
 steps
 apple-invert-lightness
 
+// Non-standard filters
+apple-blur enable-if=HAVE_CORE_MATERIAL
```

Comment:
Should be -apple- (apple-invert-lightness was a mistake)

Reply:
I originally had `-apple-blur`, but it wasn't working. Turns out that is due to `isAppleLegacyCSSValueKeyword`, which changes that into `-webkit-blur` during parsing.

I'll rename to `-apple-system-blur`, as the use of `system` will cause `isAppleLegacyCSSValueKeyword` to return false, and the keyword won't be unexpectedly transformed.